### ISSUE TITLE
Naming inconsistency: padding_mode vs pad_mode

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#39260]
[torch.stft](https://pytorch.org/docs/stable/torch.html?highlight=stft#torch.stft) uses `pad_mode`

[grid_sample](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.grid_sample) uses `padding_mode`

[nn.Conv3d](https://pytorch.org/docs/stable/nn.html?highlight=conv3d#torch.nn.Conv3d) uses `padding_mode`

[F.conv4d](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.conv3d) misses any padding mode arg altogether

**Comment by SsnL** [comment]
Can't comment on the others. But `stft` API was made to mimic `librosa`.
